packages/mobile/src/services/offline-downloader/offline-storage.ts

```diff
@@ -0,0 +1,80 @@
+import path from 'path'
+
+import type { Track } from '@audius/common'
+import RNFS, { exists } from 'react-native-fs'
+
+import { getCoverArtUri } from './offline-downloader'
+
+export const downloadsRoot = path.join(RNFS.CachesDirectoryPath, 'downloads')
+
+export const getPathFromRoot = (string: string) => {
+  return string.replace(downloadsRoot, '~')
+}
+
+export const getLocalTrackRoot = (track: any) => {
+  return path.join(downloadsRoot, `/tracks`)
+}
+
+// Track Json
+
+export const getLocalTrackJsonPath = (track: Track) => {
+  return path.join(getLocalTrackRoot(track), `${track.track_id}.json`)
+}
+
+// Cover Art
+
+export const getLocalCoverArtPath = (track: Track): [string?, string?] => {
+  const uri = getCoverArtUri(track)
+  if (!uri) return []
+  const fileName = getArtFileNameFromUri(uri)
+  return [`${downloadsRoot}/tracks/${track.track_id}`, fileName]
```

Comment:
Kinda weird that this function is returning a tuple but the function name is `getLocalCoverArtPath`